oxd-server/src/test/resources/testng.xml

```diff
@@ -5,6 +5,7 @@
     <parameter name="host" value="http://localhost"/>
     <parameter name="opHost" value="https://${test.server.name}"/>
     <parameter name="opDiscoveryPath" value=""/>
+    <parameter name="tokenAlgos" value="HS256 HS384 HS512 RS256 RS384 RS512 ES256 ES384 ES512 PS256 PS384 PS512"/>
```

Comment:
Remove it and list explicitly as commented above.

Reply:
Done